### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by DevUserNaveen**
please review it

### CODE REVIEW

File: purple.text [hunk 1]
```diff
@@ -0,0 +1 @@
+hello purple
```
Comment (naveenanimation20) [inline purple.text:1]: where is logic?

**Review by naveenanimation20 — CHANGES_REQUESTED** [review]
make changes

**Review by DevUserNaveen — COMMENTED** [review]
done

**Review by naveenanimation20 — APPROVED** [review]
approved

**Pull request merged**